db/migrate/20220729032237_add_index_to_chat_message_created_at.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddIndexToChatMessageCreatedAt < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_index :chat_messages, :created_at, algorithm: :concurrently
```

Comment:
The current discussion aside, since the `chat_messages` table is likely to be big, I'll recommend adding a `WHERE deleted_at IS NULL` filter to this index to keep it small and efficient.

Reply:
Yep have already changed it to that locally, just trying to trigger it now with a larger record set.